scripts/pylib/twister/twisterlib/testsuite.py

```diff
@@ -385,6 +386,7 @@ def load(self, data):
     def add_subcases(self, data, parsed_subcases, suite_names):
         testcases = data.get("testcases", [])
         if testcases:
+            self.has_explicit_testcases_in_yaml = True
```

Comment:
that is a veeeeeeeeeery long variable :)

Reply:
Yeah, I will figure out a new one.